index.html

```diff
@@ -1,186 +1,196 @@
+        </ul>
+        <button class="action" type="button">See Project</button>
+      </li>
+      <li class="frame fifthF">
+        <h3 class="titlePost">Profesional Art Printing Data</h3>
+        <p class="titlePostText">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="codeBtnContainer">
+          <li class="codeBtn">HTML</li>
+          <li class="codeBtn">CSS</li>
+          <li class="codeBtn">Javascript</li>
+        </ul>
+        <button class="action" type="button">See Project</button>
+      </li>
+    </ul>
+  </section>
+  <section class="aboutme">
+    <h2 class="aboutmeTitle">About me</h2>
+    <p class="aboutmeText">
+      Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work
+      and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <button class="btn" type="button">Get My Resume</button>
+    <ul class="skills">
+      <li class="skill">
+        <div class="division">
+          <div class="shape1"></div>
+          <span class="skillName">Language</span>
+        </div>
+        <ul class="childUl">
+          <li class="childLi">Javascript</li>
+          <li class="childLi">Ruby</li>
+          <li class="childLi">HTML</li>
+          <li class="childLi">CSS</li>
+        </ul>
+      </li>
+      <li class="skill">
+        <div class="division">
+          <div class="shape2"></div>
+          <span class="skillName">Frameworks</span>
+        </div>
+        <ul class="childUl">
+          <li class="childLi">Bootstrap</li>
+          <li class="childLi">Ruby on Rails</li>
+          <li class="childLi">RSPec</li>
+          <li class="childLi">CapyBara</li>
+          <li class="childLi">Selenium</li>
+        </ul>
+      </li>
+      <li class="skill">
+        <div class="division">
+          <div class="shape3"></div>
+          <span class="skillName">Skills</span>
+        </div>
+        <ul class="childUl">
+          <li class="childLi">Codekit</li>
+          <li class="childLi">Github</li>
+          <li class="childLi">Codepen</li>
+          <li class="childLi">Gitlab</li>
+          <li class="childLi">Terminal</li>
+        </ul>
+      </li>
+    </ul>
+  </section>
+  <section class="contact-form">
+    <h2 class="contact-text">
+      I'm always interested in hearing about new projects, so if you'd like to
+      chat please get in touch.
+    </h2>
+    <form class="form" action="https://formspree.io/f/mnqrnjbr" method="POST">
+      <input class="input" type="text" maxlength="30" name="name" placeholder="Full Name" required />
+      <input class="input" type="email" name="email" placeholder="Email address" required />
+      <textarea class="input" name="message" maxlength="500" placeholder="Enter text here"
```

Comment:
Excellent implementation of this section. Your portfolio and specifically the contact form design is professional and looks really good. Well done! 👏 👏